src/api/todos.ts

```diff
@@ -1,10 +1,67 @@
 import { Todo } from '../types/Todo';
 import { client } from '../utils/fetchClient';
 
-export const USER_ID = 0;
+export const USER_ID = 2248;
 
 export const getTodos = () => {
   return client.get<Todo[]>(`/todos?userId=${USER_ID}`);
 };
 
-// Add more methods here
+export const getCompletedTodos = () => {
+  return client
+    .get<Todo[]>(`/todos?userId=${USER_ID}`)
+    .then(allTodos => allTodos.filter(todo => todo.completed));
+};
+
+export const getActiveTodos = () => {
+  return client
+    .get<Todo[]>(`/todos?userId=${USER_ID}`)
+    .then(allTodos => allTodos.filter(todo => !todo.completed));
+};
```

Comment:
This logic is quite similar, filtering needs to be done on the front end so as not to make unnecessary requests.
<img width="605" alt="image" src="https://github.com/user-attachments/assets/89d03c7b-c928-451c-b2b2-bf4f64c4b294" />